integration_tests/dataproc_test_case.py

```diff
@@ -199,6 +199,14 @@ def tearDown(self):
     def getClusterName(self):
         return self.name
 
+    @staticmethod
+    def getBaseOS():
+        # Get the base operating system: '1.5-debian10' -> 'debian'
+        # Dataproc default "preview" version is Debian, so "debian" 
+        # is returned if just "preview" is provided for the image version.
+        return "debian" if FLAGS.image_version == "preview" else \
+            re.search("-[a-z]+", FLAGS.image_version).group()
```

Comment:
What it will return for `2.0.0-RC22-debian10` version?

Reply:
I think I've covered all edge-cases now, but please let me know if I haven't.